shared/js/background/debug.es6.js

```diff
@@ -9,14 +9,16 @@ const https = require('./https.es6')
 const tds = require('./storage/tds.es6')
 const startup = require('./startup.es6')
 const browserWrapper = require('./wrapper.es6')
+const utils = require('./utils.es6')
 
 self.dbg = {
     settings,
     startup,
     tabManager,
     atb,
     https,
-    tds
+    tds,
+    utils
```

Comment:
This allows us to run _dbg.utils.resetSessionKey()_ in devtools windows to verify that the session key is reset.